tests/e2e_regression/run.sh

```diff
@@ -102,11 +102,18 @@ for ((i = 0; i < nCases; i++)); do
     curl --silent --show-error --dump-header "$outDir/$name.headers" "$url" >"$outDir/$name.body"
     # Try to pretty-print and normalize (for stable diffs) the output.
     # If `jq` fails, output was probably not JSON; leave it as-is.
+    #
+    # .first_activity is ignored, because its not deterministic between fast-sync and slow-sync.
```

Comment:
Unfortunately, I had to do this. Because of the height at which we process genesis changes the results of first_activity.

Since first_activity is not the most important field here I prefer this solution to the alternative, which would have separate tests/expected results for `eden` and `edenfast`.

Reply:
why does it end up different?

Reply:
Some addresses are only visible when fetching state-to-genesis and processing it. Therefore, the earliest first_activity timestamp we have for those addresses is when this process occurs. This is because we do not process the entire chain. Otherwise, each address would either be among the static list of addresses or be seen in a transaction.

Specifically, for 'eden' and 'edenfast', the state-to-genesis occurs at different heights - since we only perform it at the beginning of slow-sync.

Reply:
does this mean real genesis plus events gives a different set of accounts from state-to-genesis?

Reply:
> does this mean real genesis plus events gives a different set of accounts from state-to-genesis?

I dont follow 100%. I dont think we fetch and process the real genesis in eden/edenfast tests, no? 

We only do StateToGenesis. In eden test, we do it at the first specified height; and in edenfast we do it at the first slow-sync height.

Reply:
oh it's different due to using `document.Time.UTC()`

thanks